src/main/java/net/sf/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -265,6 +280,14 @@ public static void openBrowser(String url) throws IOException {
         openExternalFilePlatformIndependent(fileType, url);
     }
 
+    public static void openBrowserCatchException(String url) {
+        try {
+            openBrowser(url);
+        } catch (IOException exception) {
+            LOGGER.error("Error while opening following ling with a browser: "+ url, exception);
+        }
+    }
+
```

Comment:
Every line of code which uses `openBrowser(String url)` just reacts with a status update in the lower update bar which can be easily overseen by the user.
I extracted it all to this new method which now additionally has an error popup window.